build-tools/syncpack.config.cjs

```diff
@@ -105,6 +104,19 @@ module.exports = {
 			range: "~",
 		},
 
+		{
+			label: "Ignore interdependencies on other Fluid packages within the workspace. This is needed because syncpack doesn't understand the workspace protocol.",
+			isIgnored: true,
```

Comment:
1. When [12.0.0-alpha.0](https://github.com/JamieMason/syncpack/releases/tag/12.0.0-alpha.0) is stable you should be able to remove this group (and the one on line 163 which I can't apply a comment to).
2. FYI `dependencies: ["**"]` and `packages: ["**"]` can be omitted if you prefer, but some people like to leave them in to be clearer.
3. There are some [new docs](https://jamiemason.github.io/syncpack/guide/getting-started) now which might be handy as well.

Thanks.